ads/alp/handler.js

```diff
@@ -235,7 +226,7 @@ export function getA2AAncestor(win) {
     return null;
   }
   const amp = origins[origins.length - 2];
-  if (!ampOrigins[amp] && !ampOrigins.hasOwnProperty(amp)) {
+  if (!isProxyOrigin(amp)) {
```

Comment:
@cramforce For `getA2AAncestor` with localhost, when do we proceed?
url1: http://localhost:8000/c/abc.html
url2: http://localhost:8000/v/abc.html
url3: http://localhost:8000/abc.html
Right now for this PR, `isProxyOrigin(url1)` is `true`, `isProxyOrigin(url2)` is `true` but `isProxyOrigin(url3)` is `false`. Is this desired? Or we should allow any url that starts with `http://localhost:8000`?